ctgan/data_transformer.py

```diff
@@ -139,39 +128,23 @@ def transform(self, raw_data):
 
         column_data_list = []
         for column_transform_info in self._column_transform_info_list:
-            column_data = raw_data[[column_transform_info.column_name]].to_numpy()
+            column_name = column_transform_info.column_name
+            data = raw_data[[column_name]]
             if column_transform_info.column_type == 'continuous':
-                column_data_list += self._transform_continuous(column_transform_info, column_data)
+                column_data_list.append(self._transform_continuous(column_transform_info, data))
             else:
-                assert column_transform_info.column_type == 'discrete'
-                column_data_list += self._transform_discrete(column_transform_info, column_data)
+                column_data_list.append(self._transform_discrete(column_transform_info, data))
 
         return np.concatenate(column_data_list, axis=1).astype(float)
 
     def _inverse_transform_continuous(self, column_transform_info, column_data, sigmas, st):
         gm = column_transform_info.transform
-        valid_component_indicator = column_transform_info.transform_aux
-
-        selected_normalized_value = column_data[:, 0]
-        selected_component_probs = column_data[:, 1:]
-
+        data = pd.DataFrame(column_data[:, :2], columns=list(gm.get_output_types()))
```

Comment:
I think the transformers now have a `get_output_columns` method that already returns a list based on the `output_types` property.

Reply:
The numerical transformers don't set the `OUTPUT_TYPES` attribute, so `get_output_columns` doesn't work. I think we would have to overwrite the method in RDT first if we wanted to use it here.

Reply:
Let's do this then. That method needs to work. Either overwrite it or set `OUTPUT_TYPES` during `fit` in the `NumericalTransformer`

Reply:
Sorry, I'm confused. You mean we should do this in RDT first and then change `get_output_types` to `get_output_columns ` here? If so, shouldn't we overwrite the `get_output_columns` for all other transforms in the numerical transform, since none of them set the `OUTPUT_TYPES`.

Reply:
Yeah, we can do this at a later time.